src/EmailReplyParser/Parser/EmailParser.php

```diff
@@ -141,6 +143,28 @@ public function setQuoteHeadersRegex(array $quoteHeadersRegex)
         return $this;
     }
 
+    /**
+     * @return string
+     * @since 2.7.0
```

Comment:
Ah I did not pay attention to that, usually I dont use this and I dont think we should be using this. The `CHANGELOG` file (or GitHub release) is way more expressive.

Reply:
ok, so remove the annotation and commit to `master`?